tests/python_tests/test_whisper_pipeline.py

```diff
@@ -232,6 +233,8 @@ def run_pipeline_with_ref(
     for _sample in sample:
         genai_result = run_genai(genai_pipe, _sample, generation_config, streamer)
         hf_result = run_huggingface(hf_pipe, _sample, generation_config)
+        print(f"HF result: {hf_result['text']}")
+        print(f"GenAI result: {genai_result.texts[0]}")
```

Comment:
Is it a leftover? I guess pytest's -vv should print the whole diff

Reply:
yes, debug print. Thanks for catch, removed